x/gov/src/params.rs

```diff
@@ -21,14 +26,17 @@ const DEFAULT_PERIOD: Duration = Duration::from_secs(172800); // 2 days
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct DepositParams {
-    pub min_deposit: Vec<Coin>,       // SendCoins?
+    pub min_deposit: SendCoins,
     pub max_deposit_period: Duration, // ?
 }
 
 impl Default for DepositParams {
     fn default() -> Self {
         Self {
-            min_deposit: vec![Coin::from_str("10000000stake").expect("default is valid")],
+            min_deposit: SendCoins::new(vec![
+                Coin::from_str("10000000stake").expect("default is valid")
```

Comment:
Is it from sdk? As I remember we use `uatom`

Reply:
[It's](https://github.com/cosmos/cosmos-sdk/blob/d3f09c222243bb3da3464969f0366330dcb977a8/x/gov/types/params.go#L51-L55). Think I should change to `uatom`?

Reply:
I don't know. Maybe we can use this denom.

Reply:
For now just make it "uatom". We might need a better solution in future.

Reply:
Done